DIMENSIONAL.md

```diff
@@ -0,0 +1,51 @@
+# The `dimensional` package
+
+- We use `coulomb` for unit analysis.
+- `coulomb` uses static types to represent units and this information doesn't make it to runtime.
+  - Eg:
```

Comment:
Did you mean to add an example here?

Reply:
Yes, which later made it into the next section. I will just remove this line.